src/main/java/com/github/tomakehurst/wiremock/core/WireMockApp.java

```diff
@@ -236,6 +240,8 @@ private List<RequestFilterV2> getV2StubRequestFilters() {
 
   private void loadDefaultMappings() {
     loadMappingsUsing(defaultMappingsLoader);
+    if (mappingsSourceExtensions != null)
```

Comment:
It looks like we're only loading from the sources provided via extensions, and continuing only to save to the default one.

I guess this is intentional? You probably don't want to save a stub to every source, although you might want to save it back to the one you loaded it from.

If the plan was for these to be read-only it would probably make more sense just to implement `MappingsLoader` rather than `MappingsSource`.

Reply:
Hi @tomakehurst,

We do not have any identifier in StupMapping to indicate the source from which a particular stub was loaded.

I would prefer to keep it read-only for now, If you are fine with this I will make the change to implement MappingsLoader rather than MappingsSource.

Reply:
I think that would make most sense. Seems a bit misleading to implement the save methods when there's no way they'll be called.

Reply:
changes done